setup.py

```diff
@@ -42,16 +42,18 @@
         'gui_scripts': [
             'trackma-gtk = trackma.ui.gtk:main [GTK]',
             'trackma-qt = trackma.ui.qt:main [Qt]',
-            'trackma-qt4 = trackma.ui.qt.qt4:main [Qt4]',
         ]
     },
```

Comment:
Please add these scripts to the poetry file as well.

https://python-poetry.org/docs/pyproject/#scripts

Reply:
Done